app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
- [ ] Great job adding the Users index view page. As per project requirements, you are supposed to have a Users show view page just as you did for posts. kindly add this `# app/views/users/show.html.erb`